app/components/documents/document-card-list.hbs

```diff
@@ -10,6 +10,7 @@
         @didDeleteContainer={{@didDeleteContainer}}
         @onOpenUploadModal={{@onOpenUploadModal}}
         @onAddPiece={{@onAddPiece}}
+        @isGenerated={{false}}
```

Comment:
Would remove since this is falsy by default

Reply:
I had him add this because I was thinking in function of the list decisions in de subcase route.
https://github.com/kanselarij-vlaanderen/frontend-kaleidos/blob/development/app/templates/cases/case/subcases/subcase/decision.hbs
We use the document-card-list there to show decisions (multiple if subcase was postponed)
We will probably need to have `@isGenerated={{@isGenerated}}` in this card-list template so that we can call `<Documents::DocumentCardList>` with `isGenerated`. But that would not support having both types (uploaded and generated)

Could remove this for now.